test/examples/expressions_explorer/expressions.ts

```diff
@@ -33,8 +34,23 @@ export default function ({ getService }: PluginFunctionalProviderContext) {
       });
     });
 
+    it('updates the variable', async () => {
+      const selector = `${testSubjSelector('expressionsVariablesTest')} ${testSubjSelector(
+        'testExpressionButton'
+      )}`;
+      await find.clickByCssSelector(selector);
+      await retry.try(async () => {
+        const el = await find.byCssSelector(selector);
+        const style = await el.getAttribute('style');
+        expect(style).to.contain('red');
+      });
+    });
+
     it('emits an action and navigates', async () => {
-      await testSubjects.click('testExpressionButton');
+      const selector = `${testSubjSelector('expressionsActionsTest')} ${testSubjSelector(
+        'testExpressionButton'
+      )}`;
+      await find.clickByCssSelector(selector);
```

Comment:
Maybe you can use `testSubjects` service here? Something like `testSubjects.click(...)`.